README.md

```diff
@@ -1,27 +1,174 @@
-# RowndAngular
+# Angular
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.1.1.
+## Installation
 
-## Development server
+run `npm install @rownd/angular` or `yarn add @rownd/react`
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+Make sure to add the preserveSymlinks property to the angular.json file
+```json
+{
+  "projects": {
+    "app": {
+      "architect": {
+        "build": {
+          "options": {
+            "preserveSymlinks": true,
+          }
+        }
+      }
+    }
+  }
+}
+```
 
-## Code scaffolding
+## Usage
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+The library provides a Rownd Module and Dependency Injection (Service).
 
-## Build
+In the main app.module.ts file, add the Rownd Module. Please include the @ngrx/store Module as well.
```

Comment:
Do devs need to install `@ngrx/store` first or is that a default?

Reply:
It's not a default when you install a basic angular app. I can only get the sdk to work if @ngrx/store is added to the main file